prow/plugins/hold/hold.go

```diff
@@ -47,13 +47,17 @@ func init() {
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	// The Config field is omitted because this plugin is not configurable.
-	return &pluginhelp.PluginHelp{
-			Description: "The hold plugin allows anyone to add or remove the '" + label + "' label from a pull request in order to temporarily prevent the PR from merging without withholding approval.",
-			WhoCanUse:   "Anyone can use the /hold command to add or remove the '" + label + "' label.",
-			Usage:       "/hold [cancel] | /unhold",
-			Examples:    []string{"/hold", "/hold cancel", "/unhold"},
-		},
-		nil
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: "The hold plugin allows anyone to add or remove the '" + label + "' label from a pull request in order to temporarily prevent the PR from merging without withholding approval.",
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/hold [cancel] | /unhold",
+		Description: "Prevents a PR from being merged.",
+		Featured:    true,
```

Comment:
I don't think this should be featured. Users don't need to use `/hold` to get their PRs merged so I wouldn't consider it a critically important command.

Reply:
done